vault/identity_store_util.go

```diff
@@ -44,14 +45,47 @@ func (i *IdentityStore) loadArtifacts(ctx context.Context, isActive bool) error
 		return nil
 	}
 
+	loadFuncEntities := func(context.Context) error {
+		reload, err := i.loadEntities(ctx, isActive, false)
+		if err != nil {
+			return fmt.Errorf("failed to load entities: %w", err)
+		}
+		if reload {
```

Comment:
I think `reload` is confusing because it's describing the action to take not the semantics of the return value which is "alias merges took place during load".